src/amo/components/App/index.js

```diff
@@ -205,17 +206,33 @@ export class AppBase extends React.Component<Props> {
       clientApp,
       i18n,
       isHomePage,
+      lang,
       location,
     } = this.props;
 
     const query = location.query ? location.query.q : null;
+    const i18nValues = {
+      locale: lang,
+    };
 
-    let defaultTitle = i18n.gettext('Add-ons for Firefox');
-    let titleTemplate = i18n.gettext('%s – Add-ons for Firefox');
+    let defaultTitle = i18n.sprintf(
+      i18n.gettext('Add-ons for Firefox (%(locale)s)'),
+      i18nValues,
+    );
+    let titleTemplate = i18n.sprintf(
+      i18n.gettext('%%s – Add-ons for Firefox (%(locale)s)'),
+      i18nValues,
+    );
 
     if (clientApp === CLIENT_APP_ANDROID) {
-      defaultTitle = i18n.gettext('Add-ons for Android');
-      titleTemplate = i18n.gettext('%s – Add-ons for Android');
+      defaultTitle = i18n.sprintf(
+        i18n.gettext('Add-ons for Android (%(locale)s)'),
+        i18nValues,
+      );
+      titleTemplate = i18n.sprintf(
+        i18n.gettext('%%s – Add-ons for Android (%(locale)s)'),
```

Comment:
Presumably dennis linter will catch this escaping being broken if a localizer accidentally removes one of the `%`?

Reply:
I will introduce a variable because dennis does not do anything.. `%%` is supported as regular content (https://github.com/willkg/dennis/commit/3c65a1ddca3628852c67d3c182adc63fcbaed330), thus it is subject to change.